ux.symfony.com/.twig-cs-fixer.dist.php

```diff
@@ -1,5 +1,44 @@
 <?php
 
-$config = new TwigCsFixer\Config\Config();
+use TwigCsFixer\Rules\Delimiter\BlockNameSpacingRule;
+use TwigCsFixer\Rules\File\DirectoryNameRule;
+use TwigCsFixer\Rules\File\FileExtensionRule;
+use TwigCsFixer\Rules\File\FileNameRule;
+use TwigCsFixer\Rules\Function\IncludeFunctionRule;
+use TwigCsFixer\Rules\Punctuation\TrailingCommaSingleLineRule;
+use TwigCsFixer\Rules\String\SingleQuoteRule;
+use TwigCsFixer\Rules\Variable\VariableNameRule;
+use TwigCsFixer\Rules\Whitespace\EmptyLinesRule;
+use TwigCsFixer\Rules\Whitespace\TrailingSpaceRule;
+use TwigCsFixer\Ruleset\Ruleset;
+use TwigCsFixer\Rules\Whitespace\BlankEOFRule;
+use TwigCsFixer\Standard\Twig;
+
+$uxRuleset = (new Ruleset())
+    ->addStandard(new Twig())
+    // Symfony standard (customized for Symfony UX)
+    ->addRule(new DirectoryNameRule(baseDirectory: 'templates', ignoredSubDirectories: ['bundles', 'components', 'demos/live_memory/components']))
+    ->addRule(new FileNameRule(baseDirectory: 'templates', ignoredSubDirectories: ['bundles', 'components', 'demos/live_memory/components'], optionalPrefix: '_'))
+    ->addRule(new DirectoryNameRule(case: DirectoryNameRule::PASCAL_CASE, baseDirectory: 'templates/components'))
+    ->addRule(new FileNameRule(case: DirectoryNameRule::PASCAL_CASE, baseDirectory: 'templates/components'))
+    ->addRule(new DirectoryNameRule(case: DirectoryNameRule::PASCAL_CASE, baseDirectory: 'demos/live_memory/components'))
+    ->addRule(new FileNameRule(case: DirectoryNameRule::PASCAL_CASE, baseDirectory: 'demos/live_memory/components'))
+    ->addRule(new FileExtensionRule())
+    // TwigCsFixer standard (customized for Symfony UX)
+    ->addRule(new BlankEOFRule())
+    ->addRule(new BlockNameSpacingRule())
+    ->addRule(new EmptyLinesRule())
+    ->addRule(new IncludeFunctionRule())
+    ->addRule(new SingleQuoteRule())
+    ->addRule(new TrailingSpaceRule())
+    ->addRule(new TrailingCommaSingleLineRule())
```

Comment:
Maybe some of these rules could be added to the Symfony standard but this need to be validated by a Symfony member.

Reply:
this is clearly out of the scope of this discussion 😅

Reply:
Sure